examples/take_pic_with_csc_with_canon_camera.py

```diff
@@ -5,34 +5,42 @@
 async def main():
     print("main method")
     async with salobj.Domain() as domain:
-        remote = salobj.Remote(domain=domain, name="GenericCamera", index=10)
+        remote = salobj.Remote(domain=domain, name="GenericCamera", index=1)
+        print("starting remote")
```

Comment:
any change you can replace `print` but logging?

Reply:
Done